package.json

```diff
@@ -12,7 +12,8 @@
     "start": "bundle exec jekyll serve --watch --incremental",
     "lint": "eslint js/app/ --ext=.js,.jsx",
     "test": "npm run lint",
-    "less": "lessc styles/less/main.less styles/main.css -x --source-map=styles/main.css.map"
+    "less": "lessc styles/less/main.less styles/main.css -x --source-map=styles/main.css.map",
+    "generate-sw": "workbox inject:manifest"
```

Comment:
I haven't worked with PWA before, but if I understand this correctly, we will have to run this script on every release to regenerate checksums? Is that correct? If so, that would be a problem, since right now, we don't run any scripts on release.

Reply:
To go the simplest way it's better to run `generate-sw` each time you release on prod. I'm not aware of a process so didn't add any changes to that.